cli/src/ui/utils/terminalCapabilities.ts

```diff
@@ -58,5 +61,5 @@ export function autoEnableKittyProtocol(): boolean {
  */
 export function disableKittyProtocol(): void {
 	// CSI < 1 u - Pop keyboard flags (disable)
-	process.stdout.write("\x1b[<1u")
+	process.stdout.write("\x1b[<u")
```

Comment:
<img width="1486" height="384" alt="CleanShot 2025-10-23 at 12 17 54@2x" src="https://github.com/user-attachments/assets/fcdb5823-0a2d-42b9-bd08-fde5ee2f5eed" />

Reply:
Just checking, it's idempotent? You can call it multiple times? (not sure if somehow you could get both exit and sigterm, or have autoEnableKittyProtocol be called multiple times

Reply:
I added two extra calls to check and didn't notice any issue, but it may be good to make sure we only send it once? 🤔